public/app/plugins/datasource/grafana-pyroscope-datasource/QueryEditor/QueryOptions.tsx

```diff
@@ -92,6 +93,21 @@ export function QueryOptions({ query, onQueryChange, app, labels }: Props) {
               }}
             />
           </EditorField>
+          {config.featureToggles.traceToProfiles && (
```

Comment:
Again if span and labels cannot mix this should be a separate tab and not presented in the same UI.